clean.py

```diff
@@ -48,24 +48,25 @@
         f"filter_small_docs_bytes_{i}": build_small_docs_bytes_filter(min_bytes=i) for i in [300, 1024]
     },
 }
-# Deduplication functions: function(ds: Dataset, num_proc: int, batch_size: int) -> Dataset
+# Deduplication functions and boolean to save a sample of the modifications: function(ds: Dataset, num_proc: int, batch_size: int) -> Dataset
```

Comment:
I chose this option because the concatenation function changes the size of the dataset which is not compatible with the `get_modified_documents` method.